demos/place_recognition_demo/python/place_recognition_demo.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+from pathlib import Path
+import sys
+import time
+from argparse import ArgumentParser, SUPPRESS
+
+import cv2
+import numpy as np
+
+from place_recognition_demo.place_recognition import PlaceRecognition
+from place_recognition_demo.visualizer import visualize
+
+sys.path.append(str(Path(__file__).resolve().parents[2] / 'common/python'))
+
+import monitors
+from images_capture import open_images_capture
+
+
+INPUT_SIZE = [200, 300]
```

Comment:
is it possible to get input size from model?

Reply:
Done